crispy_forms/bootstrap.py

```diff
@@ -414,3 +414,57 @@ def __init__(self, field, *args, **kwargs):
 
 class InlineField(Field):
     template = "%s/layout/inline_field.html"
+
+
+class Modal(LayoutObject):
+    """
+    Boostrap layout object for rendering crispy forms objects inside a bootstrap modal.
+
+    The following attributes can be set:
+        - `css_id`: modal's DOM id
+        - `css_class`: modal's DOM classes
+            - NOTE: "modal" and "fade" are applied by default
+        - `title`: text to display in the modal's header
+            - NOTE: text will be wrapped in a <h5> tag
+        - `title_id`: title's DOM id
+        - `title_class`: titles's DOM classes
+            - NOTE: "modal-title" is applied by default
+
+    Example::
+
+        Modal(
+            'field1',
+            Div('field2'),
+            css_id="modal-id-ex",
+            css_class="modal-class-ex,
+            title="This is my modal"
+        )
+    """
+
+    template = "%s/layout/modal.html"
+
+    def __init__(self, *fields, **kwargs):
```

Comment:
I totally agree. I would of done it how you've shown in your example from the start but I was following the signature style of other layout objects in the codebase. I personally like the defaults being defined in the signature because that way it pops up when typing out the function and it's clear what the defaults are.

Reply:
Agree, this looks great. Thanks.